libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/metrics/NavigationStepData.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation.metrics
+
+import com.mapbox.services.android.navigation.v5.internal.navigation.routeprogress.MetricsRouteProgress
+
+/**
+ * Class that contains step meta data
+ */
+internal class NavigationStepData(metricsRouteProgress: MetricsRouteProgress) {
```

Comment:
Should we `NavigationStepData` a `data class`?

Reply:
I revert making it as a data class because we failed `SchemaTest` (compare count and fields of event classes). I think when we made it like a data class - the field we pass in constructor (`metricsRouteProgress`) became a field of class.... and then when we made comparison in test - it fails.